ProiectPractica/Repository/RepositoryConfigurare.cs

```diff
@@ -10,23 +10,44 @@
 
 namespace ProiectPractica.Repository
 {
-    internal class RepositoryConfigurare : IRepositoryConfigurare
-    {  
+    public class RepositoryConfigurare : IRepositoryConfigurare
+    {
+        private string filePath;
+        public RepositoryConfigurare(string filePath) 
+        { 
+            this.filePath = filePath;
+        }
+
         /// <summary>
-        /// This function deserializes a JSON string if the string is valid and returns the configuration.
-        /// If the string fails the validation or the file does not exist it returns a null object
+        /// This function deserializes a JSON string and returns the configuration.
+        /// If the string can't be deserialized it returns a null object
         /// </summary>
         /// <param name="filePath"></param>
-        /// <returns>The configuration from the file if it is valid or a null Configurare object otherwise</returns>
-        public Configurare? GetConfigurareFromString(string filePath)
+        /// <returns>The configuration from the file, if it can be deserialized or null otherwise</returns>
+        public Configurare? GetConfigurareFromFile()
         {
-            var stringFromFile = ReadFromFile(filePath);
-            if (ValidateFileContent(stringFromFile))
+            var stringFromFile = ReadFromFile();
+            try
             {
-                 var configurare = JsonConvert.DeserializeObject<Configurare>(stringFromFile);
-                 return configurare;
+                
+                var configurare = JsonConvert.DeserializeObject<Configurare>(stringFromFile);
```

Comment:
Am sters functia aia de validare si am lasat doar deserializarea,, deoarece este mai permisiva, doar in cazul valorilor nule nu functioneaza. Sa fac atributele din Configurare nullable ca sa functioneze si cu valori nule?

Reply:
Eu zic ca nu, nu avem motiv momentan sa avem atribute nullable, vrem ca toate sa existe si sa fie prezente in UI,